src/prefect/client/schemas/actions.py

```diff
@@ -393,11 +393,11 @@ class DeploymentFlowRunCreate(ActionBaseModel):
         default_factory=objects.FlowRunPolicy
     )
     tags: list[str] = Field(default_factory=list)
-    idempotency_key: Optional[str] = Field(default=None)
-    parent_task_run_id: Optional[UUID] = Field(default=None)
-    work_queue_name: Optional[str] = Field(default=None)
-    job_variables: Optional[dict[str, Any]] = Field(default=None)
-    labels: KeyValueLabelsField = Field(default_factory=dict)
+    idempotency_key: Optional[str] = Field(None)
+    parent_task_run_id: Optional[UUID] = Field(None)
+    work_queue_name: Optional[str] = Field(None)
+    job_variables: Optional[dict] = Field(None)
+    labels: Optional[KeyValueLabelsField] = Field(default=None)
```

Comment:
I think we probably want to go back what we had before because we want to keep the `default` kwarg and more specific typing. Plus, I'm pretty sure we want labels to default to an empty dictionary.
```suggestion
    idempotency_key: Optional[str] = Field(default=None)
    parent_task_run_id: Optional[UUID] = Field(default=None)
    work_queue_name: Optional[str] = Field(default=None)
    job_variables: Optional[dict[str, Any]] = Field(default=None)
    labels: KeyValueLabelsField = Field(default_factory=dict)
```